src/kirby/components/divider/divider.component.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../scss/utils';
+
+hr {
+  border: 0px;
+  border-top: 1px solid var(--kirby-divider-color);
+
+  :host-context(.default),
```

Comment:
Usualt applied the other way around:
`:host-context(.default) hr`

Reply:
I've moved the `:host-context` out to the root level. The `hr` is placed at the root level as it needs the same properties (except from the color) for both contexts.